support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -261,11 +262,24 @@ function deriveSubsAcquisitionData(
 
 }
 
+function deserialiseAusMomentAcquisitionData(serialised: string): Object {
+  const [page, socialPlatform, referralCode] = serialised.split('_');
+  return {
+    componentId: `${page}_${socialPlatform}`,
+    source: 'SOCIAL',
+    queryParameters: {
+      referralCode,
```

Comment:
I think this needs to be structured differently?
See the `AcquisitionQueryParameters` type

Reply:
Thanks @tomrf1, I'm cross with myself for missing that. I've sorted it now and updated the PR description.